### ISSUE TITLE
[Setup] Show warning when running on docker for Win using JSON backend

### ISSUE THREAD

**Issue by orels1** [Cog-Creators/Red-DiscordBot#2989]
# Other bugs

#### What were you trying to do?

Run red through an archlinux container on windows 10 host using JSON as a backend using existing data from a previous installation

#### What were you expecting to happen?

It to work

#### What actually happened?

It crashed and burned with a cryptic error

![image](https://user-images.githubusercontent.com/3798928/64489686-a811ac00-d25e-11e9-8ead-a584fee398b9.png)

#### How can we reproduce this issue?

Run the bot under archlinux/base on a windows host using a volume connected to Windows FS as the bot data path

#### Additional info

So after talking with @mikeshardmind i tried using a second docker-specific volume as the bot data path to work around Windows being Windows.

Basically the install process looked like this then:

- `docker volume create` the actual volume i'll be using
- `docker run` connecting both newely created volume and a path to a host machine (i used `-v <localPath>:<containerPath>` to mount)
- install all the things inside the container
- install the bot and do a first launch
- shut down the bot and copy over the data from the host-connected volume into the docker-specific volume
- launch the bot, now it should work fine

Hope this info helps, sharing since Sinbad asked me to, would love to see some warning about windows issues in the docs or instead of that crashlog

**Comment by mikeshardmind** [comment]
I can repro this on WSL version 1, but not with WSL version 2. Given that and what I observed about the failure condition, I don't think this is something we can fix, but it could be documented that WSL v2 is needed for WSL users.
